actors/migration/nv16/test/miner_migration_test.go

```diff
@@ -146,11 +150,48 @@ func createMinersAndSectorsV7(t *testing.T, ctx context.Context, ctxStore adt.St
 	return append(minersToProve, minerInfos...), v
 }
 
+func makeManifest(t *testing.T, store cbor.IpldStore) cid.Cid {
+	adtStore := adt.WrapStore(context.Background(), store)
+	builder := cid.V1Builder{Codec: cid.Raw, MhType: mh.IDENTITY}
+
+	manifestData := manifest8.ManifestData{}
+	for _, name := range []string{"system", "init", "cron", "account", "storagepower", "storageminer", "storagemarket", "paymentchannel", "multisig", "reward", "verifiedregistry"} {
+		codeCid, err := builder.Sum([]byte(fmt.Sprintf("fil/8/%s", name)))
```

Comment:
paranoia: maybe make it something slightly different to make sure we're not running into usage of the old cid pattern: "fil/8/test/%s"

Reply:
that breaks the state invariant checks, we will probably want to use them in future migrations.

Reply:
Not sure I understand I feel like that's exactly the kind of thing I was hoping to flush out.  We should fix the state invariant checks to work with the correct CIDS / parameterize them for testing.  We use state invariant checks on main net state not just in testing.

Reply:
wait, this code will never run again after this migration.

Or do you want to keep around the state checking for post migration offline work?

Reply:
so what do you want me to do here?

Reply:
also note, we can read the state of the system actor to find the actual code cids, so no need to do any extra parameterization.

Reply:
Could @vyzo or @ZenGround0 post a link to the state invariant checks we're talking about? In builtin-actors we use [static `/fil/test/<type>` code CIDs](https://github.com/filecoin-project/builtin-actors/blob/5396be01b74f8a90517f31e4db2c0f13f85a3f4c/actors/runtime/src/test_utils.rs#L35-L46) to refer to test actors. This isn't quite the same situation since we're dealing with a CodeCID migration here, so may want to have a "before" and an "after" set? However, without knowing which checks you're talking about, I can't really give an opinion here.

Reply:
@raulk https://github.com/filecoin-project/specs-actors/blob/master/actors/states/check.go#L28

Reply:
> Or do you want to keep around the state checking for post migration offline work?

@vyzo the state invariant checks are used [here](https://github.com/filecoin-project/ent).  I'm now realizing this should migrate to use what we will build in builtin actors since specs-actors is retiring.  I'll add a comment to the builtin actors issue.